trieste/acquisition/interface.py

```diff
@@ -37,6 +37,18 @@
 with a batch dimension, i.e. an input of shape `[..., 1, D]`.
 """
 
+VectorizedAcquisitionFunction = Callable[[TensorType], TensorType]
+"""
+Type alias for vectorized acquisition functions.
+
+An :const:`VectorizedAcquisitionFunction` differs from an :const:`AcquisitionFunction` only by its
+output shape. An :const:`VectorizedAcquisitionFunction` maps a set of maps a set of `B` query points
```

Comment:
Should we use (here and everywhere else) another letter for the vectorised dimension? It should be entirely possible to vectorise batch acquisition functions, i.e. you would obtain a batch of size `V*B` by optimising V batches of size B. Even if we never do this, I think it would avoid a lot of confusion.

Reply:
done